### PR TITLE
docs: Pre-translation Queue, Request New Language, CLS, Crowdin Apps …

### PR DESCRIPTION
**Pull request by rohalskyy**
…JS, etc.

### CODE REVIEW

File: src/content/docs/crowdin/translation-process/pre-translation.mdx [hunk 1]
```diff
@@ -68,6 +68,20 @@ To apply Pre-translation manually, follow these steps:
...
 
+### Pre-translation Queue
+
+The Pre-translation queue allows you to trigger multiple manual pre-translations via TM, MT, or AI, each with different settings. After initiating a pre-translation, each instance enters a queue with a status of **Pending**.
+
+Upon triggering a manual pre-translation, you will receive a notification indicating that the pre-translation has been added to the queue. To review all pre-translations currently in the queue, navigate to **Tools > Pre-translation queue** in your project. Alternatively, click **Pre-translation** in the project's **Dashboard** and select **Queue** to be redirected directly to the **Tools > Pre-translation queue**.
+
+Each entry in the Pre-translation queue table includes the following details:
+
+- **Created at**
```
Comment (andrii-bodnar) [inline src/content/docs/crowdin/translation-process/pre-translation.mdx:79]: This option with no description looks strange, please consider adding some description

File: src/content/docs/developer/modules/ui-modules/custom-ai.mdx [hunk 2]
```diff
@@ -112,7 +112,9 @@ For Crowdin Enterprise:
...
+Crowdin sends prompts to the app using `chatCompletionsUrl`, and the app then processes these prompts and responds to Crowdin with a reply. To enable your Custom AI as an assistant in the Crowdin Editor, the app must implement the `stream` parameter using [server-sent events](https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events#event_stream_format) for real-time interaction via streaming responses. When streaming is supported, it allows AI-generated content to be delivered progressively, providing real-time feedback to users. This improves the workflow by offering immediate suggestions or corrections without waiting for a complete response, ensuring a smooth, interactive experience for tasks like confirming terminology, checking consistency, or refining translations. Crowdin includes the `stream` parameter set to `true` in requests to the app for `chatCompletions` and expects a response in [the corresponding format](#expected-response-from-the-app-streaming).
```
Comment (andrii-bodnar) [inline src/content/docs/developer/modules/ui-modules/custom-ai.mdx:115]: This part is too big and unstructured. I'd suggest splitting it into smaller paragraphs. For example:

```
Crowdin sends prompts to the app using `chatCompletionsUrl`, and the app then processes these prompts and responds to Crowdin with a reply. 

To enable your Custom AI as an assistant in the Crowdin Editor, the app must implement the `stream` parameter using [server-sent events](https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events#event_stream_format) for real-time interaction via streaming responses.

When streaming is supported, it allows AI-generated content to be delivered progressively, providing real-time feedback to users. This improves the workflow by offering immediate suggestions or corrections without waiting for a complete response, ensuring a smooth, interactive experience for tasks like confirming terminology, checking consistency, or refining translations.

Crowdin includes the `stream` parameter set to `true` in requests to the app for `chatCompletions` and expects a response in [the corresponding format](#expected-response-from-the-app-streaming).
```

Comment (andrii-bodnar) [inline src/content/docs/developer/modules/ui-modules/custom-ai.mdx:115]: > To enable your Custom AI as an assistant in the Crowdin Editor, the app must implement the `stream` parameter

Isn't this optional? The app could work in the no-streaming mode by providing the "Expected Response from the App" response.

**Review by andrii-bodnar — CHANGES_REQUESTED** [review]

File: src/content/docs/crowdin/translation-process/pre-translation.mdx (hunk 1 above)
Comment (rohalskyy) [reply src/content/docs/crowdin/translation-process/pre-translation.mdx:79]: Added a description

File: src/content/docs/developer/modules/ui-modules/custom-ai.mdx (hunk 2 above)
Comment (rohalskyy) [reply src/content/docs/developer/modules/ui-modules/custom-ai.mdx:115]: Applied your suggestion

Comment (rohalskyy) [reply src/content/docs/developer/modules/ui-modules/custom-ai.mdx]: Updated texts according to the optionality of the `stream` parameter

**Review by andrii-bodnar — APPROVED** [review]

**Pull request merged**